ui/outbox.lua

```diff
@@ -5,7 +5,8 @@ local S = minetest.get_translator("mail")
 function mail.show_sent(name, sortfieldindex, sortdirection, filter)
     sortfieldindex = tonumber(sortfieldindex or mail.selected_idxs.sortfield[name]) or 3
     sortdirection = sortdirection or mail.selected_idxs.sortdirection[name] or "1"
-	filter = filter or ""
+	filter = mail.selected_idxs.filter[name] or filter or ""
```

Comment:
Same as previous comment. Not sure if this needs any changes now but probably should be fixed at some point, I guess it might be needed because of current formspec logic.
So again if actually needed then better to leave it and not start rewrite on this pull request.

Reply:
Seems you've already updated these 👍